test/utils/Checkpoints.test.js

```diff
@@ -0,0 +1,31 @@
+const { expectRevert } = require('@openzeppelin/test-helpers');
+
+const { expect } = require('chai');
+
+const CheckpointsImpl = artifacts.require('CheckpointsImpl');
+
+contract('Checkpoints', function (accounts) {
+  beforeEach(async function () {
+    this.checkpoint = await CheckpointsImpl.new();
+    this.tx1 = await this.checkpoint.push(1);
+    this.tx2 = await this.checkpoint.push(2);
+    this.tx3 = await this.checkpoint.push(3);
```

Comment:
We need to check the behavior of `getAtBlock` for a block that did not have a checkpoint stored. For this we should add a line somewhere between pushes using `time.advanceBlock` from Test Helpers, and then add a new case to the `getAtBlock` test below.